packages/rocketchat-message-star/client/views/starredMessages.js

```diff
@@ -0,0 +1,69 @@
+/*globals StarredMessage */
```

Comment:
Can you delete the file in your commit? Or do you want to keep the old .coffee file?